apps/admin/backend/src/util/get_current_time.ts

```diff
@@ -0,0 +1,8 @@
+/**
+ * Simply a wrapper for `Date.now()`, returns the current time in Unix time.
+ *
+ * Separated out into its own file so that it can be mocked in tests.
```

Comment:
Using `jest` to mock the date doesn't work for some reason?

Reply:
It made more sense on the frontend because, when using fake timers, the time advances within test even if you mock it to begin with. So you have to remember to mock the date before every render. But on the backend, when renders are more controlled, could probably just go back to `jest`.